### ISSUE TITLE
parse Verification status and Confidentiality request in the swrs_history schema

### ISSUE THREAD

**Issue by matthieu-foucault** [bcgov/cas-ggircs#458]
### Description

Requested by Qinghan. 

TODO:
- [ ] clarify the value and urgency of this feature

**Comment by LindsayMacfarlane** [comment]
Thank you for creating this ticket @matthieu-foucault - I have followed up with Chris to get a better understanding of the value/need for this request.

**Comment by dleard** [comment]
@LindsayMacfarlane Is this something we plan to do?

**Comment by LindsayMacfarlane** [comment]
Good catch @dleard. I will follow up with Emilie to see if this is still needed.

**Comment by LindsayMacfarlane** [comment]
@dleard - can you remind me the context of this request?

**Comment by dleard** [comment]
I don't remember, I think Matthieu was the contact for this request. It just sounds like there is more data in the XML reports that they would like parsed by the ETL. But that's all the context I have.

**Comment by LindsayMacfarlane** [comment]
I followed up with ES and she did not remember the context of this request either. I'll continue to dig through past emails, but given it hasn't come up as an issue since June, I'm pulling the ticket back into the icebox.
